src/chrome/skin/icons/freebsd/hicolor/22x22/apps/chatzilla.png

```diff
@@ -0,0 +1 @@
+../../../../img/chatzilla22.png
```

Comment:
On 02/03/2015 18:55, foudfou wrote:
>
> In src/chrome/skin/icons/freebsd/hicolor/22x22/apps/chatzilla.png 
> <https://github.com/foudfou/FireTray/pull/167#discussion_r25657639>:
>
> > @@ -0,0 +1 @@
> > +../../../../img/chatzilla22.png
>
> As long as the content does not differ, I guess symlinking directories 
> would be best.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/foudfou/FireTray/pull/167/files#r25657639>.
>
This then pans into the conversation earlier about using symlinks in the 
xpi.
Would we use symlinks in the source and let the make build the actual paths?
s